enterprise-beans/4.0/_index.md

```diff
@@ -7,18 +7,25 @@ summary: "Release for Jakarta EE 9"
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record]()
-  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```

Comment:
Here's a pointer to the EJB 4.0 release record which can be inserted into line 9:  https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan

Reply:
There's still no useful text in the release record, is there supposed to be?

Reply:
This text is now updated.

Reply:
There's still no link here.

Reply:
Good catch.  Should be there now.